lib/wor/batchifier/exceptions.rb

```diff
@@ -5,6 +5,7 @@ class WrongStrategy < StandardError; end
       class ExistingStrategy < StandardError; end
       class InterfaceNotImplemented < StandardError; end
       class StrategyNotFound < StandardError; end
+      class ProcessingError < StandardError; end
```

Comment:
I'm still not %100 convinced with this name for the exception. This should be a total giveaway that the merge strategy defined on a custom strategy has failed for some reason, I'll keep thinking about the name for this exception.

Reply:
Maybe something in the lines of "CustomStrategyMergingError"

Reply:
@holywyvern what you think about this?